### DISCUSSION TITLE
Adding the word "MOD" to the Title on the Nexus Page

### DISCUSSION THREAD

**Discussion by offron** [piotrulos/MSCModLoader#170] [General]
By instinct new players probably look for "Mod" or "Loader"
Problem with that is that all the results point to the outdated "Mod Loader Pro"

If the Title would include the Word "Mod" or "Loader" i assume the MSCLoader would pop up much higher.
An easy solution could already be to write the title like: "MSC Loader" with a Space between

Why? 
First, it seems that ALOT of ppl are either not aware or too lazy to read. 
I have to admit that when i started using mods years ago, i fell into the trap of "MLP" aswell.

Also to mitigate the potential damage the "PRO Loader" does. 
It feels like that min 50% of reports of "non working" mods by users iis the fact that they use "MLP".
That introduces Spam in Reddit / Discord or on the Page itself where the Mod is hosted.

Im pretty sure this could remove Overhead for the Dev's and Mods to write redundant messages like: "This mod doesnt work with MLP, use MSC Loader"

![mod keyword 2](https://github.com/piotrulos/MSCModLoader/assets/80930618/31d4209b-9fe4-49d4-81a2-acfbc0861353)
![mod keyword 3](https://github.com/piotrulos/MSCModLoader/assets/80930618/105ff277-0442-4c16-90d8-dddce99df090)
![mod keyword 1](https://github.com/piotrulos/MSCModLoader/assets/80930618/0140238e-ab3b-492a-b218-5220a0733816)

**Comment by piotrulos** [comment]
The issue isn't name, it was named mscloader since beggining, the issue is that if you search on youtube how to install mods for msc, some youtuber made tutorial like 2 years ago and was using pro. So basically most people that downloads pro comes from that tutorial.

All people (who actually bother to answer) when I asked "why are you using pro" was poining to youtube tutorial.    
I think only way to stop people from downloading would be to somehow contact nexus staff so they could marked it as outdated/abandoned.

**Reply by offron** [reply]
yeah i remember it was named the same since ever. I was thinking it would finding it easier and faster for new ones.
Also, if ppl try to look for the "pro" they should find the "MSC Loader" in the list aswell. Piggybacking from it.
Not the solution but probably a work around.

In the end its 100% your choice if u wanna rename the mod, i was just thinking it could help the newbies in the community.

Thats actually a good idea. You have probably the bigger reach when it comes to Nexus staff, is that something you could do?

**Reply by piotrulos** [reply]
Never talked to nexus staff, so idk.

**Reply by offron** [reply]
the only contact information i found was "support@nexusmods.com" If you wanna give it a shot that would be great, otherwise let me know and il try my luck

Edit: Is there a chance that MLP stole sourcecode from you that you could file a DMCA? 
I assume u might have done that already, but cant hurt to ask. 
Im not sure however how it works as you you have the Mod als GLP3 (i have no idea what the diffrences are)
